README.md

```diff
@@ -1 +1,13 @@
-# vagrant-is
\ No newline at end of file
+# WSO2 Identity Server Vagrant Resources
+
+This repository contains the following resources:
+
+- Vagrant box generation
+
+Resources for creating Vagrant boxes are contained here. Users can define box specifications
```

Comment:
Change descriptions according to above and indent properly.

Reply:
+1